### PR TITLE
Andsemenov/setup fake applicant account

### PR DESCRIPTION
**Pull request by andsemenov**
Added  role  VARCHAR(15) NOT NULL in table users. It might be useful to reduce symbols for columns in the table in the future.

### CODE REVIEW

File: server/db/populate-db.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
 -- Users seed data
 -- Note: Do NOT add passwords you use in real life. Passwords are now saved as clear text. This is very bad from security point of view. We will have a task to encrypt the passwords
-INSERT INTO users (email, password) values ('admin@cyf.org', 'admin_password');
-INSERT INTO users (email, password) values ('user@cyf.org', 'user_password');
+INSERT INTO users (email,role, password) values ('admin@cyf.org', 'applicant','admin_password');
+INSERT INTO users (email, role, password) values ('user@cyf.org', 'applicant','user_password');
```
Comment (bonboh) [inline server/db/populate-db.sql]:
```suggestion
INSERT INTO users (email, role, password) values ('admin@cyf.org', 'applicant', 'admin_password');
INSERT INTO users (email, role, password) values ('user@cyf.org', 'applicant', 'user_password');
```

Some software teams care about consistent code style, and one part of this is ensuring that there is always a space after a comma. For this project, I don't think this needs to be mandatory but it's always good to keep it in mind when writing and reviewing your code!

File: server/db/recreate-schema.sql [hunk 2]
```diff
@@ -5,5 +5,6 @@ DROP TABLE if exists users;
 CREATE TABLE users (
   id SERIAL PRIMARY KEY,
   email    VARCHAR(200) NOT NULL,
+  role     VARCHAR(15) NOT NULL
```
Comment (bonboh) [inline server/db/recreate-schema.sql]:
```suggestion
  role     VARCHAR(15) NOT NULL,
```
It looks like we're missing a comma at the end of this line. It's good practise to test your changes before requesting review!

**Review by bonboh — CHANGES_REQUESTED** [review]
Good work @andsemenov — I've left a couple of comments for you

File: server/db/recreate-schema.sql (hunk 2 above)
Comment (andsemenov) [reply server/db/recreate-schema.sql]: Hi Coung, I've changed the code.

**Review by bonboh — APPROVED** [review]
Looks good to me!

**Pull request merged**